apps/sensenet/src/components/dialogs/upload/upload-dialog.tsx

```diff
@@ -164,6 +166,17 @@ export function UploadDialog(props: UploadDialogProps) {
     }
   }
 
+  const customUpload = async () => {
+    setIsUploadInProgress(true)
+    try {
+      await props.customUploadFunction?.(files, progressObservable)
+    } catch (error) {
+      logger.error({ message: 'Upload failed', data: error })
+    } finally {
+      setIsUploadInProgress(false)
+    }
+  }
+
```

Comment:
do not create a new function to ovverride upload request, something like this would be enough:
`customUploadFunction ? customUploadFunction(files, progressObservable) : await repository.upload.fromFileList....`